src/schemas/integrator_acl_schema.json

```diff
@@ -0,0 +1,28 @@
+{
+    "id": "/integratorAclSchema",
+    "properties": {
+        "apiKeys": {
+            "items": {
+                "type": "string"
+            },
+            "type": "array"
+        },
+        "integratorId": {
+            "type": "string"
+        },
+        "label": {
```

Comment:
is this integrator name?

Reply:
yep.. maybe not the best naming but it's already there https://github.com/0xProject/0x-api/blob/master/src/schemas/api_key_schema.json#L8